descriptors/ModuleDescriptor-template.json

```diff
@@ -56,13 +59,19 @@
           "methods": ["GET"],
           "pathPattern": "/query",
           "permissionsRequired": ["fqm.query.sync.get"],
-          "modulePermissions": ["perms.users.get"]
+          "modulePermissions": [
+            "perms.users.get",
+            "consortia.user-tenants.collection.get"
+          ]
         },
         {
           "methods": ["GET"],
           "pathPattern": "/query/{query-id}",
           "permissionsRequired": ["fqm.query.async.results.get"],
-          "modulePermissions": ["perms.users.get"]
+          "modulePermissions": [
+            "perms.users.get",
+            "consortia.user-tenants.collection.get"
```

Comment:
Are we going to explode anything if this is deployed in non-ECS?

Reply:
Good question, I'm honestly not sure how it would handle that ECS permission in non-ECS environments, I couldn't find much documentation on it. We do need that permission for ECS environments though. Maybe it makes more sense to put it in permissionsDesired, that way it should be ok with the permission not being present.